portfolio/src/main/webapp/script.js

```diff
@@ -92,30 +103,28 @@ function deleteAllComments() {
  * Transforms a comment object to a list element styled with UI Kit classes
  */
 function transformCommentToListElement(comment) {
-  const identiconUrls = [
-    'images/identicon-1.png',
-    'images/identicon-2.png',
-    'images/identicon-3.png',
-    'images/identicon-4.png',
-    'images/identicon-5.png',
-    'images/identicon-6.png',
-    'images/identicon-7.png',
-    'images/identicon-8.png',
-  ];
-  // Pick a random identicon to use as an avatar
-  const avatarUrl =
-      identiconUrls[Math.floor(Math.random() * identiconUrls.length)];
-
   const listElement = document.createElement('li');
-  litHtml.render(commentTemplate(comment, avatarUrl), listElement);
+  litHtml.render(commentTemplate(comment), listElement);
 
   return listElement;
 }
 
 /**
- * Constructs a comment template from a comment object and an avatar url
+ * Deletes a comment when its associated delete button is clicked
+ */
+function deleteCommentOnClick() {
```

Comment:
This name is a little misleading, as you're not really deleting a comment by calling this method, but installing event listeners to delete comments when the buttons are clicked.